plugins/scheduled_task/batch/ScheduledTaskRunner/KScheduledTaskRunner.class.php

```diff
@@ -143,6 +158,12 @@ protected function getSortedScheduledTaskProfiles($maxProfiles = 500)
 		$filter->lastExecutionStartedAtLessThanOrEqualOrNull = strtotime('today UTC');
 		$pager = new KalturaFilterPager();
 		$pager->pageSize = $maxProfiles;
+		
+		$runnerTypes = $this->getAdditionalParams(KScheduledTaskRunner::runnerTypes);
```

Comment:
this is not on the template file
do we need both runnerTypes and params.recyclebin? looks overlapping